workshops/boost-research-reproducibility-binder/paper.md

```diff
@@ -0,0 +1,113 @@
+
+# Statement of Need
+
+Reproducible research is necessary to ensure that scientific work can be trusted. 
+Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. 
+The goal is to ensure that all results can be independently verified and built upon in future work, which is sometimes easier said than done. 
+Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. 
+
+*[The Turing Way](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)* is a project [funded](https://www.turing.ac.uk/news/alan-turing-institute-spearhead-new-cutting-edge-data-science-and-artificial-intelligence) as part of UKRI’s Strategic Priorities fund. 
+It is an open source handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do". 
+The goal is to provide all the information that researchers need at the start of their projects to ensure that they are easy to reproduce at the end. 
+The [handbook](https://the-turing-way.netlify.com) currently includes chapters and training material on reproducibility, open research, version control, collaborating on GitHub/GitLab, research data management, reproducible environments, analysis testing, reviewing, continuous integration, reproducible research with Make and risk assessment. 
+Further chapters in development include coding style for reproducibility, credit for reproducible research and reproducible data analysis pipelines for machine learning, and will build upon Alan Turing Institute case studies. 
+The project demonstrates open and transparent project management and communication with future users, as it is openly developed at the [GitHub repository](https://github.com/alan-turing-institute/the-turing-way). 
+It also facilitates workshops to train researchers and research software engineers to use tools to make their work more reproducible - one of which is the *Boost your research reproducibility with Binder* workshop.
+
+The Binder project significantly lowers the bar to sharing and re-using software, as it allows a user to try out someone else's work simply by clicking a single link. 
+The majority of researchers and educators have not yet discovered the Binder project, or they have heard it mentioned but do not know how to get started [@build_a_binder]. 
+It is also a tool that requires some investment to master. 
```

Comment:
Hmm, if you know what you're doing, setting up a Binder takes 10 minutes. Could we reword this to something around "the tool is most impactful if some time is invested to understand its potential and workings" or something similar?

Reply:
👍